templates/index.html

```diff
@@ -0,0 +1,13 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My site</title>
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}" />
```

Comment:
upewnij sie w konsoli, ze to czyta, bo z ustawien wynika, ze django w ogole tych staticow nie serwuje